fileholder.go

```diff
@@ -21,6 +21,9 @@ const (
 type FileWriter interface {
 	io.Writer
 
+	// Close the output stream. If writter is not closable, it returns nil.
+	Close() error
```

Comment:
Tohle je jednoznačně hezká myšlenka. Vůbec nechápu, proč jsem to tak nenapsal. Asi jsem měl nějaké zatmění :-)